### ISSUE TITLE
SimBridge: Add timeouts to all client calls

### ISSUE THREAD

**Issue by tracernz** [flybywiresim/a32nx#7594]
### Aircraft Version

Development

### Description

#7593 adds a fetchWithTimeout function. Expand it's use to the rest of the SimBridge functions.

**Comment by Vinitvh** [comment]
Hi, @tracernz. I would like to work on this.

**Comment by MaverickDe** [comment]
Hi, can i get assigned to this issue,Thanks

**Comment by tracernz** [comment]
After #7593 has merged into the `master` branch, it can be done. Basically need to look through https://github.com/flybywiresim/a32nx/tree/master/src/simbridge-client for any fetch calls... in some cases it might be necessary for a longer timeout than default e.g. the terrain map calls. There is a param for that in the new function.

**Comment by juliansebline** [comment]
@Vinitvh @MaverickDe I suggest you implement it on your side and create a PR

**Comment by Vinitvh** [comment]
@tracernz Okay.

**Comment by Vinitvh** [comment]
@juliansebline Looks like some functions don't need `fetchWithTimeout` function like `getImageList` in Viewer.ts. Do you want me to keep `fetch` for those functions or replace all with timeout function?
